package.json

```diff
@@ -28,7 +28,7 @@
     "localforage": "1.7.2",
     "localforage-observable": "1.4.0",
     "marked": "github:mattermost/marked#ed33baecd7d7fa97d479ba22dde9d226b083d67d",
-    "mattermost-redux": "github:mattermost/mattermost-redux#40199d90bc867e0d0265ece6f84b1199c82a07f0",
+    "mattermost-redux": "github:uber-uchat/mattermost-redux#sidebar-fixed",
```

Comment:
Nothing to change here for now obviously, but this is a reminder that we'll need to update this before merge